src/com/yaji/Datehandler.java

```diff
@@ -0,0 +1,23 @@
+package com.yaji;
+
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Datehandler {
```

Comment:
I understand why you made this class, but it is not really necessary. Firstly we do not use it anywhere. A good principle is KISS: Keep it Simple, Stupid. Secondly, it is such a trivial wrapper, that you could simply write `DateFormat dateFormat = new SimpleDateFormat("d.M.y"); Date newDate = dateFormat.parse("2.12.2008")` when you need it. 

You can leave it for now.